inspire_utils/name.py

```diff
@@ -87,6 +87,10 @@ def __init__(self, name, constants=None):
             if not self._parsed_name.first:
                 self._parsed_name.first = self._parsed_name.title
                 self._parsed_name.title = ''
+            if not self._parsed_name.last:
+                self._parsed_name.last = self._parsed_name.first
+                self._parsed_name.first = self._parsed_name.title
+                self._parsed_name.title = ''
```

Comment:
Hm are you sure it will cover all the cases when last name is empty? For example what if there will be first but not the title? Maybe it's worth then to add title to the if condition?